includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -178,23 +176,27 @@ private static function normalizeControllerClass( $controllerClass, &$controller
 	/**
 	 * Generates a list of possible template names ordered by preference
 	 *
-	 * @param string $controllerClass
+	 * @param string $controllerClassWithNamespace
 	 * @param string $methodName
 	 * @param string $controllerBaseName (optional) save cpu cycles by not
 	 *
 	 * @return array
 	 */
-	private function getTemplateOptions( $controllerClass, $methodName, $controllerBaseName = null ) {
+	private function getTemplateOptions( string $controllerClassWithNamespace, string $methodName, $controllerBaseName = null ): array {
 		$templates = [];
 
-		$fromAnnotation = $this->getTemplateAnnotation( $controllerClass, $methodName );
+		$controllerClass = $this->getClassName( $controllerClassWithNamespace );
+
+		$fromAnnotation = $this->getTemplateAnnotation( $controllerClassWithNamespace, $methodName );
 		if ( !empty( $fromAnnotation ) ) {
 			$templates[] = $fromAnnotation;
 		}
 
 		// Add variations on the controller name
 		if ( $controllerBaseName === null ) {
-			$controllerBaseName = F::app()->getBaseName( $controllerClass );
+			$controllerBaseName = $this->getClassName(
+				F::app()->getBaseName( $controllerClass )
```

Comment:
Here `$controllerClass` was already passed through `WikiaView::getClassName`, so the namespaces should be gone. Should we pass it again?